concepts/0003-protocols/template.md

```diff
@@ -93,7 +93,7 @@ the matrix form is used in many early RFCs. We leave it up to
 the community to settle on whether it wants to strongly recommend specific
 diagram types.
 
-The formal names for each state are important, as they are used in [`ack`s]( https://github.com/hyperledger/indy-hipe/pull/77)
+Regardless of the type of diagram that's used, [the formal names for each state are important](https://docs.google.com/presentation/d/17hk6QqLW5M9E4TBPZwXIUBu9eEinMNXReceyZTF4LpA/edit?usp=sharing), as they are used in [`ack`s]( https://github.com/hyperledger/indy-hipe/pull/77)
```

Comment:
This reference is to a HIPE PR. Pretty sure it should be to the Aries RFC.

Reply:
I'll fix these. However, I'd like to point out that I didn't introduce the links to HIPEs; they were present in the doc I was editing, and I just left them in. Unfortunately, the github diff makes them look like I added them.

I wonder if there's a good way to find other examples of this problem, so we can gradually drive it out of the corpus...

Reply:
@dhh1128 understand about the HIPE links being there already.

I'm wondering if these changes should go into the new 0000-template-protocol.md and we delete this file?  I was going to go back and fix these, but I think we can just eliminate the file.

What do you think?

Reply:
In my latest push into this PR, I have largely removed the old doc. I kept a stub so we could retain the diagram of structure, but everything else is gone. Also, all references to it have been shifted. I had to move some more detail about adopted messages into the template. Tell me what you think, @swcurran .